Comparing source compatibility of E:\Java\.m2\repository\org\apache\jspwiki\jspwiki-bootstrap\2.12.3\jspwiki-bootstrap-2.12.3.jar against E:\Java\.m2\repository\org\apache\jspwiki\jspwiki-bootstrap\2.12.2\jspwiki-bootstrap-2.12.2.jar
WARNING: You are using the option '--ignore-missing-classes', i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.bootstrap.WikiBootstrapServletContextListener  (not serializable)
	===  CLASS FILE FORMAT VERSION: 55.0 <- 55.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC WikiBootstrapServletContextListener()
	===  UNCHANGED METHOD: PUBLIC void contextDestroyed(javax.servlet.ServletContextEvent)
	===  UNCHANGED METHOD: PUBLIC void contextInitialized(javax.servlet.ServletContextEvent)

Semantic versioning suggestion: 0.0.1